modules/getbooks.js

```diff
@@ -0,0 +1,3 @@
+export default function getBooks() {
```

Comment:
-  [ ] As i suggested above, I think you should use the ES6 arrow function, I'll suggest you follow the requirement and refactor these functions as arrow functions is a compact alternative to a traditional function expression, with some semantic differences:+1: :smile:

Reply:
thank you @V-Blaze . change done!